diff --git a/Library/src/entities/AnimatedEntity.cpp b/Library/src/entities/AnimatedEntity.cpp
--- a/Library/src/entities/AnimatedEntity.cpp
+++ b/Library/src/entities/AnimatedEntity.cpp
@@ -306,11 +306,13 @@ std::vector<Renderable> AnimatedEntity::Render()
     {
         Renderable item;
         item.type = RenderableType::SOLID_CS;
-        item.model = glMatrixFromTransform(getOTransform());
+        item.model = glMatrixFromTransform(getOTransform() * T_O2C);
         items.push_back(item);

         if(graObjectId >= 0)
         {
+            item.model = glMatrixFromTransform(getOTransform()
+                                               * (dm == DisplayMode::GRAPHICAL ? T_O2G : T_O2C));
             item.type = RenderableType::SOLID;
             item.cor = glVectorFromVector(getOTransform().getOrigin());
             item.vel = glVectorFromVector(getLinearVelocity());
